### DISCUSSION TITLE
Software that looks interesting?

### DISCUSSION THREAD

**Discussion by sr320** [sr320/ceabigr#5] [General]
Please add links to any software that might relevant.. whether used or not. 

Specifically looking for programs designed for multiomics analysis.

**Comment by sr320** [comment]
via @yaaminiv https://thackl.github.io/gggenomes/

**Comment by sr320** [comment]
via @AHuffmyer https://strimmerlab.github.io/software/corpcor/

This package implements a James-Stein-type shrinkage estimator for the covariance matrix, with separate shrinkage for variances and correlations. The details of the method are explained in Schäfer and Strimmer (2005) and Opgen-Rhein and Strimmer (2007). The approach is both computationally as well as statistically very efficient, it is applicable to "small n, large p" data, and always returns a positive definite and well-conditioned covariance matrix. In addition to inferring the covariance matrix the package also provides shrinkage estimators for partial correlations and partial variances. The inverse of the covariance and correlation matrix can be efficiently computed, as well as any arbitrary power of the shrinkage correlation matrix. Furthermore, functions are available for fast singular value decomposition, for computing the pseudoinverse, and for checking the rank and positive definiteness of a matrix.

**Comment by AHuffmyer** [comment]
Multi-omic analysis with MiBiOmics: https://gitlab.univ-nantes.fr/combi-ls2n/mibiomics. 

"MiBiOmics is a shiny-based web application to perform correlation network analysis and multi-omics analysis on omics datasets. MiBiOmics is be available at https://shiny-bird.univ-nantes.fr/app/Mibiomics but can also be runned locally with a conda environment or with docker. To run MiBiOmics locally you need to install *MiBiOmics.yml* environment."

**Comment by hputnam** [comment]
Parsimonious Gene Correlation Network Analysis (PGCNA): a tool to define modular gene co-expression for refined molecular stratification in cancer https://www.nature.com/articles/s41540-019-0090-7

**Comment by ksil91** [comment]
RaMWAS R package: A complete toolset for methylome-wide association studies (MWAS). It is specifically designed for data from enrichment based methylation assays, but can be applied to other data as well. Additionally, RaMWAS include tools for joint analysis of methlyation and genotype data. I used RaMWAS to integrate my SNP data into the genome to extract CpG-SNPs.
https://bioconductor.org/packages/release/bioc/html/ramwas.html

**Comment by AHuffmyer** [comment]
MiMeNet: Exploring Microbiome-Metabolome Relationships using Neural Networks

GitHub: https://github.com/YDaiLab/MiMeNet
Paper: https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009021#sec011

**Comment by laurahspencer** [comment]
@ksil91 and have experimented with [MACAU](https://doi.org/10.1371/journal.pgen.1005650), which could theoretically test for associations between a variable of interest (e.g. expression of particular genes) and methylation status, while controlling for relatedness (using a relatedness matrix).

**Comment by laurahspencer** [comment]
[mixOmics](http://mixomics.org/) offers a wide range of multivariate methods for the exploration and integration of biological datasets with a particular focus on variable selection

Vignette: https://mixomicsteam.github.io/Bookdown/index.html

**Comment by AHuffmyer** [comment]
Thanks for posting @laurahspencer ! I was just looking at MixOmics in more detail. The [DIABLO method](http://mixomics.org/mixdiablo/) does correlations between multi-omic datasets. [Here is a paper that used DIABLO](https://www.nature.com/articles/s41598-021-94096-6) to correlate microbiome and metabolome data in a coral larvae/CCA study.

**Comment by shellywanamaker** [comment]
https://www.omicsanalyst.ca/

**Comment by yaaminiv** [comment]
https://bioconductor.org/packages/release/bioc/vignettes/GEM/inst/doc/user_guide.html
